docs/supporting/faq.md

```diff
@@ -104,6 +104,6 @@ From **File** > **Preferences** > **User Settings**, add the following option to
 
 ## Technical Support
 
-You can ask questions and search for answers on [Stack Overflow](https://stackoverflow.com/questions/tagged/vscode), provide suggestions on [UserVoice](http://go.microsoft.com/fwlink/?LinkID=533482), and enter issues directly in our [GitHub repository](https://github.com/Microsoft/vscode/issues).
+You can ask questions and search for answers on [Stack Overflow](https://stackoverflow.com/questions/tagged/vscode) and enter issues and feature requests directly in our [GitHub repository](https://github.com/Microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions).

```

Comment:
What's the best link for users wanting to submit issues?
In the blog, you point to contributing.md.
"If your request is not on this list or if you have a new request, please open a new GitHub issue following these [guidelines](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md)"